discovery-infra/start_discovery.py

```diff
@@ -345,16 +345,19 @@ def _cluster_create_params():
         "vip_dhcp_allocation": bool(args.vip_dhcp_allocation) and not user_managed_networking,
         "additional_ntp_source": ntp_source,
         "user_managed_networking": user_managed_networking,
-        "high_availability_mode": "None" if args.master_count == 1 else "Full"
+        "high_availability_mode": "None" if args.master_count == 1 else "Full",
+        "olm_operators": list(map(lambda name: {'name': name}, utils.get_env('OLM_OPERATORS', '').lower().split()))
```

Comment:
nit: `list(map(lambda name: ` can be rephrased with list comprehension more concisely:

`"olm_operators": list(map(lambda name: {'name': name}, utils.get_env('OLM_OPERATORS', '').lower().split()))`

to

`"olm_operators": [{'name': name} for name in utils.get_env('OLM_OPERATORS', '').lower().split()]`

Reply:
Also see https://github.com/openshift/assisted-test-infra/pull/707/files#r614728967